Exercise image downloader should only fetch accepted exercises

gmmoraes opened the pull request:
…s:2'

#363
Changed the url at the "exercise_api" variable to search only exercises with "status:2"

rolandgeider commented on wger/exercises/management/commands/download-exercise-images.py:
```diff
@@ -67,9 +67,12 @@ def handle(self, **options):
 
-        exercise_api = "{0}/api/v2/exercise/?limit=999"
+        #Adding 'status=2' at the end of the exercise_api url will filter the json data for only accepted exercises 
+        exercise_api = "{0}/api/v2/exercise/?limit=999;status=2"
```
shouldn't this be ``?limit=999&status=2``?

gmmoraes replied:
- Changes:

    Substituted the semicolon for an ampersand at the "exercise_api" variable.

I think that both can be used as query separators since I was getting the same results with the semicolon and ampersand. 

https://wger.de/api/v2/exercise/?limit=999;status=2
https://wger.de/api/v2/exercise/?limit=999&status=2

gmmoraes commented:
Changes:
- Substituted the semicolon for an ampersand at the "exercise_api" variable.


I was getting the same results with the semicolon and ampersand, but using the ampersand seems to be better.

https://wger.de/api/v2/exercise/?limit=999;status=2
https://wger.de/api/v2/exercise/?limit=999&status=2

gmmoraes commented:
Ok!
I updated the code and removed the comments.

2017-04-13 19:43 GMT-03:00 Roland Geider <notifications@github.com>:

> *@rolandgeider* requested changes on this pull request.
>
> Can you remove the comments or convert them to logs? Then I can merge this
> ------------------------------
>
> In wger/exercises/management/commands/download-exercise-images.py
> <https://github.com/wger-project/wger/pull/373#discussion_r110271368>:
>
> >          image_api = "{0}/api/v2/exerciseimage/?exercise={1}"
>          thumbnail_api = "{0}/api/v2/exerciseimage/{1}/thumbnails/"
> +
> +        #exercise_counter = 0  Uncomment this in order print in the console application the number of exercises being downloaded.
>
> No need leave this commented, you can use the --verbosity flag
> <https://docs.djangoproject.com/en/1.10/ref/django-admin/#cmdoption-verbosity>
> and print it if it's set to 2 or higher
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wger-project/wger/pull/373#pullrequestreview-31434548>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APoDgacz7gXxWrpC-VS4bgOXcSd4JdDQks5rvqUPgaJpZM4MxEls>
> .
>

rolandgeider commented:
Merged
